sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -1044,7 +1044,10 @@ directive:
 - from: swagger-document
   where: $.parameters.BlobPublicAccess
   transform: $.required = true;
-```
+- from: swagger-document
+  where: $.definitions.ContainerProperties
+  transform: $.required.push("PublicAccess");
```

Comment:
Why is `PublicAccess` required?  Doesn't `default(PublicAccess)` become `None`?

Reply:
This is needed to make the property non-nullable in the ContainerProperties model.